src/fstext/trivial-factor-weight.h

```diff
@@ -52,17 +52,8 @@
 // This has the advantage that it always works, for any input (also I just
 // prefer this approach).
 
-#ifdef _MSC_VER
 #include <unordered_map>
 using std::unordered_map;
```

Comment:
I am guessing there are other places in the code base with similar include guards which are no longer necessary since unordered map is in std namespace in all c++11 standard libraries. I'll update utils/stl-utils.h.